src/components/Concept.tsx

```diff
@@ -0,0 +1,256 @@
+import Error from '@theme/Error';
```

Comment:
delete this

Reply:
Resolved with [08842b0](https://github.com/EtherealEngine/etherealengine-docs/pull/156/commits/08842b05237dae37e4c9b8ab7d754e07b4a0da77)